openacademy/tests/create_objects.py

```diff
@@ -0,0 +1,31 @@
+def create_course(model, name, description, responsible_id):
+    """Create and return a course object."""
+    course = model.create(dict(
```

Comment:
Instead of dict, pass  an actual  dictionary, i.e. `.create({...`

Reply:
Could you apply this same comment in all places, please?

Reply:
Got it.